vector/src/main/res/layout/fragment_login_server_url_form.xml

```diff
@@ -55,7 +55,7 @@
 
             <com.google.android.material.textfield.TextInputLayout
                 android:id="@+id/loginServerUrlFormHomeServerUrlTil"
-                style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense.ExposedDropdownMenu"
+                style="@style/Widget.Material3.TextInputLayout.OutlinedBox.Dense.ExposedDropdownMenu"
```

Comment:
This was a mistake to directly use `Widget.MaterialComponents.*` in the layout. We should always have override style to our `style_*.xml`. Can I ask you to create the missing files, or add style to existing files following the current naming convention? Thanks!
It apply to all the other changes in layout files.